.storybook/addons.js

```diff
@@ -1,2 +1,4 @@
-import '@kadira/storybook/addons';
-import '@kadira/storybook-addon-options/register';
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
```

Comment:
This is the default storybook config as per the slow start guide. The reason is that these modules are included as part of the `@storybook/react` package and not explicitly defined in our package.json